### ISSUE TITLE
List of notifications 

### ISSUE THREAD

**Issue by colinking** [peacecorps/BonVoyage#92]
To track what situations will send notifications to different BonVoyage users, I thought it would be useful to create an exhaustive list of all cases where a notification may be sent, what communication channels it is sent along, and who would receive it.

**Comment by colinking** [comment]
@SeanBae are staff/volunteers notified when they are assigned to a request? If not we should add an issue for that.

**Comment by seanbae** [comment]
@colinking Yes both staff and volunteers are currently notified via SMS when a new request is created.
They are also notified via SMS when a new comment is made (which includes when a request is updated).

**Comment by seanbae** [comment]
We currently are looking to centralize all the notification feature and integrate the HTML5 Notifications API.

**Comment by choquette33** [comment]
I noticed today that when I was assigned as a reviewer, I did not receive and sms notification. Is that a bug or is that action not part of the sms notification list.

**Comment by seanbae** [comment]
@choquette33 I will go take a look this weekend.
